substrate/frame/society/src/lib.rs

```diff
@@ -1382,6 +1388,66 @@ pub mod pallet {
 			// if backend == 0 { return Err(Error::<T, I>::NoVotes.into()); };
 			Ok(Pays::No.into())
 		}
+
+		/// Poke / reconsider the deposit reserved when bidding.
```

Comment:
```suggestion
		/// Update the deposit reserved when bidding.
```
I think we can keep it simple for the reader, but no strong opinion.

Reply:
I have no strong opinion either, if I make another commit, I will update this as well.